spec/duration.html

```diff
@@ -1244,23 +1246,28 @@ <h1>RoundDuration ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _sec
           1. Let _fractionalSeconds_ be _nanoseconds_ × 10<sup>−9</sup> + _microseconds_ × 10<sup>−6</sup> + _milliseconds_ × 10<sup>−3</sup> + _seconds_.
         1. Let _remainder_ be *undefined*.
         1. If _unit_ is *"years"*, then
+          1. Let _dateAdd_ be ? GetMethod(_calendar_, *"dateAdd"*).
```

Comment:
is it intentional to throw this error here, before the CreateTemporalDuration ones, and not at the time it’s first invoked?

Reply:
It has to be in some defined order, but it doesn't have to be this particular order. I can move it up to just before it's first invoked.